Source/WebKit/WebProcess/InjectedBundle/API/glib/WebKitWebExtension.cpp

```diff
@@ -20,6 +20,8 @@
 #include "config.h"
 #include "WebKitWebExtension.h"
 
+#include <WebKit/WKBundleAPICast.h>
```

Comment:
We're already in WebKit, so move this down below and use "" style includes instead: `#include "WKBundleAPICast.h"`